### ISSUE TITLE
typeID duplicates for items 

### ISSUE THREAD

**Issue by ignkhorlan** [evepraisal/go-evepraisal#101]
Hi there,

my original problem was that evepraisal does return a wrong typeID for "Avatar", it returns 50097 instead of 11567 (example: [Avatar evepraisal](https://evepraisal.com/a/mhzlj.json) vs. [correct typeID](https://www.fuzzwork.co.uk/api/typeid.php?typename=Avatar)).

After some further research, I found that if you search for "Avata" via the search bar you get two entries for an Avatar and some duplicates for other items as well ([example](https://evepraisal.com/search?q=Avata)). Fuzzwork's [typeID.csv](https://www.fuzzwork.co.uk/resources/typeids.csv) seems to have duplicate entries for "Avatar" as well, I guess one is the actual item and one is a Collidable Structure or something.

tl;dr:
**Expected behavior:** Evepraisals use the items that can actually be traded/refined/used
**Actual Behavior:** Evepraisals use the "wrong" item sometimes
**Example:** https://evepraisal.com/a/mhzlj

**Comment by sudorandom** [comment]
Interesting problem. Maybe adding the type along with the name might help. Or just sort non-market items lower.

**Comment by sudorandom** [comment]
This should be fixed now.
